### PR TITLE
Override only appear in table and resets when being put back in req bar

### PR DESCRIPTION
**Pull request by CalderLund**

### CODE REVIEW

File: src/components/Cards/CourseCard.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
               <v-tooltip top open-delay="300" max-width="350px" v-if="this.courseData.overridden">
                   <template v-slot:activator="{ on, attrs }">
-                    <v-chip  color="red" label x-small text-color="white" class="chip" v-bind="attrs" v-on="on">
+                    <v-chip label x-small text-color="white" class="chip" v-bind="attrs" v-on="on">
```
Comment (haowei99) [inline src/components/Cards/CourseCard.vue:55]: what color is it then?

File: src/components/Modals/RequirementOptionsModal.vue [hunk 2]
```diff
@@ -157,6 +157,7 @@
                 } else {
                     if (!this.onSelectionBar) {
                         this.course.inRequirementBar = true;
+                        this.course.overridden = false;
```
Comment (haowei99) [inline src/components/Modals/RequirementOptionsModal.vue:160]: need to also hide major and minor chip in the requirement bar. Reset them to default

**Review by haowei99 — CHANGES_REQUESTED** [review]

File: src/components/Cards/CourseCard.vue (hunk 1 above)
Comment (CalderLund) [reply src/components/Cards/CourseCard.vue:55]: It is light grey. Looks a lot better imo.

Comment (CalderLund) [reply src/components/Cards/CourseCard.vue:55]: The default colour.

File: src/components/Modals/RequirementOptionsModal.vue (hunk 2 above)
Comment (CalderLund) [reply src/components/Modals/RequirementOptionsModal.vue:160]: We don't store the chip info there. By setting `inRequirementBar = true` we effectively do that.

Comment (CalderLund) [reply src/components/Modals/RequirementOptionsModal.vue:160]: A lot of the cases you want address are in the other PR.

Comment (haowei99) [reply src/components/Modals/RequirementOptionsModal.vue:160]: sounds good

File: src/components/Cards/CourseCard.vue (hunk 1 above)
Comment (haowei99) [reply src/components/Cards/CourseCard.vue:55]: kk

**Review by haowei99 — APPROVED** [review]

**Pull request merged**